src/inmanta/moduletool.py

```diff
@@ -369,7 +370,7 @@ def v1tov2(self, module: str) -> None:
         """
         module = self.get_module(module)
         if not isinstance(module, ModuleV1):
-            raise ModuleVersionException(f"Expected a v1 module, but found v{module.GENERATION.value} module")
+            raise ModuleVersionException(f"Expected a v1 module, but found v{module.ModuleGeneration.value} module")
```

Comment:
this looks odd?

Reply:
I did a search and replace for a typo and it looks like I didn't pay sufficient attention, thanks.